libdnf/dnf-sack.cpp

```diff
@@ -2147,7 +2150,9 @@ process_excludes(DnfSack *sack, GPtrArray *enabled_repos)
         }
     }
     
```

Comment:
... and trailing whitespaces here too (as well as four lines above this line). Could you take care of it? (it's a good idea to configure your editor to highlight trailing whitespaces... if possible)